raiden/api/python.py

```diff
@@ -220,21 +219,25 @@ def token_network_register(
         if not is_binary_address(token_address):
             raise InvalidBinaryAddress("token_address must be a valid address in binary")
 
+        if token_address == NULL_ADDRESS_BYTES:
+            raise ValueError("token_address must be non-zero")
+
         # The following check is on prestate because the chain state does not
         # change here.
         # views.state_from_raiden() returns the same state again and again
         # as far as this gevent context is running.
         if token_address in self.get_tokens_list(registry_address):
             raise AlreadyRegisteredTokenAddress("Token already registered")
 
+        chainstate_before_addition = views.state_from_raiden(self.raiden)
```

Comment:
What does the renaming achieve here now?There is no other chainstate in the function. Better keep it as was if there is no benefit to the change.